core/test/feed.test.ts

```diff
@@ -111,3 +111,18 @@ test('shows that user has any feeds', async () => {
   await deleteFeed(id)
   equal(hasFeeds.get(), false)
 })
+
+test('change feed and post reading status', async () => {
+  let feedId = await addFeed(testFeed())
+  let feed = getFeed(feedId)
+  let posts = getPosts()
+  keepMount(feed)
+  keepMount(posts)
+  await addPost(testPost({ feedId }))
+
+  equal(ensureLoaded(feed.get()).reading, 'fast')
+  equal(ensureLoaded(posts.get()).list[0]?.reading, 'fast')
+  await changeFeed(feedId, { reading: 'slow' })
```

Comment:
Can you add filter to the test?

Let’s we have 2 old posts (one with type by filter, other with default type).